wallet-api/src/Wallet/Emulator/Http.hs

```diff
@@ -192,13 +167,46 @@ walletHandlers state = hoistServer api (runM state) $ walletApi :<|> controlApi
     controlApi =
       blockchainActions :<|> setValidationData :<|> blockValidated :<|>
       blockHeight
+    assertionsApi = assertOwnFundsEq :<|> assertIsValidated
+
+assertOwnFundsEq ::
+     (MonadError ServantErr m, MonadReader State m, MonadIO m)
+  => Wallet
+  -> Value
+  -> m NoContent
+assertOwnFundsEq wallet value =
+  fmap (const NoContent) $ runTrace $ Types.assertOwnFundsEq wallet value
```

Comment:
`fmap (const x) y` is just `(x <$ y)` or `(y <$ x)`

Reply:
nice